gensim/models/atmodel.py

```diff
@@ -882,6 +882,84 @@ def get_document_topics(self, word_id, minimum_probability=None):
 
         raise NotImplementedError('Method "get_document_topics" is not valid for the author-topic model. Use the "get_author_topics" method.')
 
+    def get_new_author_topics(self, corpus, minimum_probability=None):
+        """Infers topics for new author.
```

Comment:
please use numpy-style docstrings: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html and https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt